vud.routing.yml

```diff
@@ -5,3 +5,22 @@ vud.admin_advanced_settings:
     _form: \Drupal\vud\Form\VudAdminAdvancedSettings
   requirements:
     _permission: 'administer vote up/down'
+
+vud.vote:
+  path: /vote/{entityTypeId}/{entityId}/{voteValue}
```

Comment:
Tokens needs to be re-added to the URL.
Tokens are a way to avoid CSRF attacks, so let's re-add it, `drupal_get_token()` got converted into `\Drupal::csrfToken()` in D8.